SR2019/2019-05-15-kit-chase-round-1.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Teams with outstanding kit
+subject: Outstanding kit
+---
+
+Hello!
+
+According to our records, you still have a kit from this years competition. As the competition has ended, we would like this back!
+
+Please securely pack the kit and ship it to the address listed on [our website](https://studentrobotics.org/contact/). Once you have dispatched it, please [let us know](mailto:teams@studentrobotics.org).
+
+## How to ship your kit
+
+- Kits should be shipped in the while 18l Really Useful Box (RUB).
+- All parts of the kit should be suitably protected either in Jiffy bags or bubble wrap.
+- Damaged batteries must not be shipped.
+- Any empty space in the box should be filled with paper or bubble wrap. No packing peanuts; They're messy.
+
+Kit data for couriers:
```

Comment:
This section reads a bit weirdly. Could we reword it a bit so it's less like some notes?

Particularly:
- the weight line contains internal notes, I think we can just say "6kg"
- the value line might be a surprise since we've not mentioned insurance elsewhere in the email (maybe we should introduce that separately earlier and keep that line here?)

Reply:
> maybe we should introduce that separately

Any idea how you imagine wording this in. I feel like simply dropping "oh BTW this kit is super expensive" is a bit odd.

Reply:
Maybe rework the line:

> Kit should be shipped using a courier which supports tracking, to ensure the kit isn't lost.

something like:

> Please insure the kit during shipping and use a courier which supports tracking, to ensure the kit isn't lost.

and then maybe move that line above the data bullets?